Dalamud/Game/Addon/Events/AddonEventManager.cs

```diff
@@ -133,7 +143,7 @@ internal void AddPluginEventController(string pluginId)
         if (this.pluginEventControllers.All(entry => entry.PluginId != pluginId))
```

Comment:
What if `this.pluginEventControllers.Add` get called from `OnFrameworkUpdate`, while this `.All(...)` is being evaluated? Might as well blindly add to `this.newEventControllers`, and let `OnFrameworkUpdate` deal with duplicates.